src/components/Pages/index.js

```diff
@@ -150,9 +151,32 @@ const VmCreatePageConnected = connect(
   })
 )(VmCreatePage)
 
+class VmConsolePage extends React.Component {
+  render () {
+    const { vms, match } = this.props
+    if (match.params.id && vms.getIn(['vms', match.params.id])) {
+      return <VmConsole consoleId={match.params.console_id} vmId={match.params.id} />
+    }
+    return null
+  }
+}
+
+VmConsolePage.propTypes = {
+  match: RouterPropTypeShapes.match.isRequired,
+  vms: PropTypes.object.isRequired,
+}
+
+const VmConsolePageConnected = connect(
+  (state) => ({
+    vms: state.vms,
+  }),
+  (dispatch) => ({})
```

Comment:
You can leave the `dispatch` function bit off and only specify the `(state) => {}` function.

Reply:
We are using this form everywhere in code, so I'm leaving it for keeping consistency.